config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,24 @@
+# rubocop:disable Lint/PercentStringArray
+SecureHeaders::Configuration.default do |config|
+  config.csp = {
+    base_uri: %w['self'],
+    block_all_mixed_content: true,
+    child_src: %w[www.youtube.com],
+    default_src: %w['self'],
+    form_action: %w['self'],
+    frame_ancestors: %w['none'],
+    img_src: %w['self' data: www.google-analytics.com *.tile.openstreetmap.org],
+    plugin_types: %w[application/x-shockwave-flash],
+    script_src: %w['self' 'unsafe-inline' dap.digitalgov.gov www.google-analytics.com],
+    style_src: %w['self' 'unsafe-inline']
+  }
+
+  config.hsts = 'max-age=15768000'
+  config.referrer_policy = %w[origin-when-cross-origin strict-origin-when-cross-origin]
```

Comment:
I wasn't familiar with [the Referrer Policy spec](https://w3c.github.io/webappsec-referrer-policy/), but I _like it_.